docs/src/operators/maintenance.md

```diff
@@ -13,3 +13,9 @@ We increment our Gardener dependency version by version following the Gardener u
 !!! info
 
     If you use the Gardener integration of metal-stack do not skip any patch releases. You may skip patch releases if you depend on metal-stack only, but we recommend to just deploy every patch release one by one for the best possible upgrade experience.
+
+## Rollback
+
+Rollback in metal-stack is possible as long as no database schema migration has occurred between the releases. The system uses forward-only DB migrations (e.g. for RethinkDB), so if a schema change is involved, reverting the application requires also restoring the database to its previous state (pre-migration). This ensures data consistency and system stability.
+
+If no database migration has taken place, the release version can simply be rolled back and redeployed. However, if a migration has occurred, the backup-restore-sidecar must be used to [manually](https://github.com/metal-stack/backup-restore-sidecar/blob/master/docs/manual_restore.md) restore the database from a backup created before the schema change.
```

Comment:
I am tempted to say that we do not test rollbacks and rollbacks are therefore not supported. 

I mean, theoretically it could be possible to do if you really know what you do. But estimating the side-effects is close to impossible for every release. It's not just the RethinkDB that's involved. If we updated a Postgres database in a release then also just starting again an older version will not work with the backup-restore-sidecar. Probably there are more things like that and certainly it's safer to just rollback a single component that is causing the problem and not an entire release.